eng/DocsGenerator/Program.cs

```diff
@@ -10,7 +11,24 @@ public class Program
 {
     public static void Main(string[] args)
     {
-        var templatePath = GetRelativeToGitRoot("eng/DocsGenerator/README.template.md");
+        var failIfChanged = args.Contains("FailIfChanged=true");
+
+        var resources = new (string Template, string Output)[]
+        {
+            ("eng/DocsGenerator/README.template.md", "README.md"),
+            ("eng/DocsGenerator/GettingStarted.template.md", "docs/AzureDevOps/GettingStarted.md"),
+            ("eng/DocsGenerator/DefinitionReference.template.md", "docs/AzureDevOps/DefinitionReference.md"),
```

Comment:
Maybe later, but this would be cool if these were args too.
Maybe like this `templatePath:destinatioPath`:
```
eng/DocsGenerator/README.template.md:README.md
```

And for local runs, we'd have a script which would just `dotnet run` this project with the same args

Reply:
@premun when would you want to run just one of the files? This runs really fast